PICMI_Python/fields.py

```diff
@@ -209,10 +209,10 @@ class PICMI_Cartesian1DGrid(_ClassWithInit):
         Position of the node at the upper bound [m]
 
     lower_boundary_conditions: vector of strings
-        Conditions at lower boundaries, periodic, open, dirichlet, or neumann
+        Conditions at lower boundaries, periodic, open, dirichlet, silver_mueller, or neumann
```

Comment:
Let me know if we shall use the long name:
```suggestion
        Conditions at lower boundaries, periodic, open, dirichlet, absorbing_silver_mueller, or neumann
```

Reply:
Ok, now I see this. I think the long name is better since says what is does, "absorbing". Is there a reference that can be added to the documentation?

Reply:
We had a good description here: https://github.com/ECP-WarpX/WarpX/pull/1766 by @RemiLehe: https://people.nscl.msu.edu/~lund/uspas/scs_2016/lec_adv/A1b_EM_Waves.pdf